android-client-sdk/src/main/java/com/devcycle/sdk/android/model/Event.kt

```diff
@@ -1,20 +1,26 @@
 package com.devcycle.sdk.android.model
 
-import com.fasterxml.jackson.annotation.JsonInclude
 import com.fasterxml.jackson.annotation.JsonProperty
+import com.google.gson.annotations.SerializedName
 import java.math.BigDecimal
 import java.util.*
 
-@JsonInclude(JsonInclude.Include.NON_NULL)
 internal data class Event private constructor(
+    @SerializedName("type")
```

Comment:
How can we test that all these JSON serialization changes are working before release?

Reply:
both unit test & manual testing have covered them.. both in minified mode & normal, uncompressed mode! we're seeing the inbound events as expected and all other API calls / responses being parsed correctly